### PR TITLE
feat(vulnerabilities): handle medium and unknown severities

### PR DESCRIPTION
**Pull request by setchy**
## Changes

Handles two additional vulnerability scenarios
- UKNOWN
- MEDIUM (same as MODERATE)

Changed validated against: https://github.com/setchy/renovate-demo-osv-severity-fix

## Context

#22239

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [x] Both unit tests + ran on a real repository

### CODE REVIEW

**Comment by setchy** [comment]
I did a search for other uses of `MODERATE` in the codebase and found
https://github.com/renovatebot/renovate/blob/f86fa3af8f0cfeb6c4c22390bbbeb39f511ccb0b/lib/types/vulnerability-alert.ts#L10-L15

Interest in input on whether we leave this as-is, or update `severity`

**Review by secustor — DISMISSED** [review]

**Review by viceice — COMMENTED** [review]
failed tests, otherwise LGTM

File: lib/workers/repository/process/vulnerabilities.ts [hunk 1]
```diff
@@ -566,7 +564,7 @@ export class Vulnerabilities {
     vulnerability: Osv.Vulnerability,
     affected: Osv.Affected
   ): SeverityDetails {
-    let severityLevel: string | undefined;
+    let severityLevel = 'Unknown';
```
Comment (secustor) [inline lib/workers/repository/process/vulnerabilities.ts:567]: Shouldn't be this capitalised? 
```suggestion
    let severityLevel = 'UNKNOWN';
```

Comment (setchy) [reply lib/workers/repository/process/vulnerabilities.ts:567]: That said, let me improve this.

Comment (setchy) [reply lib/workers/repository/process/vulnerabilities.ts]: Added a titleCase utility function and made severityLevel always uppercased

**Review by JamieMagee — APPROVED** [review]

**Pull request merged**